build.gradle

```
@@ -3,32 +3,22 @@ plugins {
 }
 
 project.ext.springdoc = '1.2.32'
+project.ext.jupiter = '5.6.0'
+project.ext.bootSecurity = '2.2.5.RELEASE'
+project.ext.springSecurity = '5.3.0.RELEASE'
 
 allprojects {
-    apply plugin: 'io.spring.dependency-management'
 
-    repositories {
-        jcenter()
-        maven { url 'https://jitpack.io' }
-    }
-
-    dependencyManagement {
-        overriddenByDependencies = false
-
-        dependencies {
-            dependency "org.springdoc:springdoc-openapi-security:${springdoc}"
-        }
-    }
 }
 
 subprojects {
     apply plugin:'java'
 
     dependencies {
-        testImplementation 'org.junit.jupiter:junit-jupiter-api'
-        testImplementation 'org.junit.jupiter:junit-jupiter-params'
+        testImplementation "org.junit.jupiter:junit-jupiter-api:${jupiter}"
+        testImplementation "org.junit.jupiter:junit-jupiter-params:${jupiter}"
 
-        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+        testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${jupiter}"
```

```suggestion
        testImplementation 'org.junit.jupiter:junit-jupiter'
```
and you can remove other jupiter deps